### PR TITLE
Feature/brands

### PR DESCRIPTION
**Pull request by andrsweb**
Creating brands section with mobile responsive

### CODE REVIEW

File: src/scss/common/grid.scss [hunk 1]
```diff
@@ -16,32 +16,19 @@
 		max-width: $container-xl
 	}
 
+	&.l {
+		@media all and (min-width: $md) {
+			max-width: $container-md
+		}
```
Comment (andreistez) [inline src/scss/common/grid.scss]: Duplicate of strings 7-9, .container already has such @media rule.

File: src/scss/common/vars.scss [hunk 2]
```diff
@@ -3,15 +3,17 @@ $xs	: 375px;
 $sm	: 480px;
 $m  : 576px;
 $md	: 768px;
+$l	: 888px;
```
Comment (andreistez) [inline src/scss/common/vars.scss]: Please, use common shorthand, e.g.: xxs, xs, sm, md, lg, xl, xxl, xxxl

Comment (andreistez) [reply src/scss/common/vars.scss]: One-letter variables naming is a bad habit.

File: src/scss/sections/brands/brands.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        width: calc( 100% / 2);
+        margin-bottom: 20px;
+        padding: 0 28px;
+
+        a:hover {
+			svg path {
+				fill: #6188ff;
+                opacity: 1;
+			}
+        }
```
Comment (andreistez) [inline src/scss/sections/brands/brands.scss]: Please, check tabs spacing here. Looks weird, maybe it's GitHub bug.

File: src/scss/sections/brands/brands.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        justify-content: center;
+        align-items: center;
+        padding-top: 20px;
+
+        @media all and (min-width: $md) {
+            padding-top: 0;
+        }
+    }
+
+    .brand {
```
Comment (andreistez) [inline src/scss/sections/brands/brands.scss]: Are you sure that .brand ___MUST___ be inside .brands?

File: src/scss/sections/brands/brands.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        display: flex;
+        justify-content: center;
+        width: calc( 100% / 2);
+        margin-bottom: 20px;
+        padding: 0 28px;
+
+        a:hover {
+			svg path {
+				fill: #6188ff;
+                opacity: 1;
```
Comment (andreistez) [inline src/scss/sections/brands/brands.scss]: Why `opacity` is here?

File: src/scss/sections/brands/brands.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+			}
+        }
+
+        svg path {
+            fill: #d6dae5;
```
Comment (andreistez) [inline src/scss/sections/brands/brands.scss]: To make hover more smooth, you can use:
```
transition: fill ease-out 0.35s
```

**Review by andreistez — CHANGES_REQUESTED** [review]

File: src/scss/sections/brands/brands.scss (hunk 3 above)
Comment (andrsweb) [reply src/scss/sections/brands/brands.scss]: it's git hub bug

File: src/scss/sections/brands/brands.scss [hunk 7]
```diff
@@ -14,32 +14,33 @@
...
+    margin-bottom: 20px;
+    padding: 0 28px;
 
         a:hover {
-			svg path {
-				fill: #6188ff;
+            svg path {
+                transition: $tr;
+                fill: #6188ff;
                 opacity: 1;
```
Comment (andreistez) [inline src/scss/sections/brands/brands.scss]: Move to link, not to svg path

File: src/scss/sections/brands/brands.scss [hunk 8]
```diff
@@ -14,32 +14,33 @@
...
+    justify-content: center;
+    width: calc( 100% / 2);
+    margin-bottom: 20px;
+    padding: 0 28px;
 
         a:hover {
-			svg path {
-				fill: #6188ff;
+            svg path {
+                transition: $tr;
```
Comment (andreistez) [inline src/scss/sections/brands/brands.scss]: Why is it here? Where you need to place transition? Why only `$tr`? What about `fill ease-out`?

**Review by andreistez — CHANGES_REQUESTED** [review]

**Review by andreistez — APPROVED** [review]

**Pull request merged**